File: Loops_Hristo_Chipev/Palindrome.cs
using System;
namespace CSharp_Tasks
{
    public class Palindrome
    {
        public static bool palindrome(string str)
        {
            bool res = false;
            string rev = "";

            rev = string.Join("", str.ToString().Split("").Reverse());

            if (rev == str)
            {
                res = true;
            }

            return res;
        }

        public static bool palindrome2(int n)
        {
            bool res = false;
            int copyOfN = n;
            string rev = "";

            while (copyOfN > 0)
            {
                rev += copyOfN % 10;
                copyOfN = copyOfN / 10;
            }

            if (int.Parse(rev) == n)
            {
                res = true;
            }

            return res;
        }

    }
}

File: Loops_ValentinArnautski/palindrome.js
function palindrome(num) {
    let isPalindrome = false;
    let reversedNumber = num.toString().split('').reverse().join('');
  
    if (num.toString() === reversedNumber) {
      isPalindrome = true;
    }
  
    console.log(isPalindrome);
  }
  
  palindrome(123321);